CHANGES.rst

```diff
@@ -1,30 +1,31 @@
 Changes
 =======
 
-Version 0.9.0 (UNRELEASED)
+Version 0.9.0 (2022-12-14)
 --------------------------
 
-- Adds kubernetes maximum memory limit, kubernetes default memory limit and the maximum workspace retention period to ``info`` endpoint.
-- Adds validation of REANA specification to ``start_workflow`` endpoint.
-- Adds interactive sessions out-of-sync check to ``reana-admin check-workflows`` command.
-- Adds workspace retention rules validation to ``get_workspace_retention_rules`` function.
+- Adds new ``/api/launch`` endpoint that allows running workflows from remote sources.
+- Adds new ``get_workflow_retention_rules`` endpoint that allows to retrieve the workspace file retention rules of a workflow.
 - Adds ``queue-consume`` command that can be used by REANA administrators to remove specific messages from the queue.
 - Adds configuration environment variable to set an API rate limit for slow endpoints (``REANA_RATELIMIT_SLOW``).
-- Adds new ``/api/launch`` endpoint that allows running workflows from remote sources.
-- Adds the possibility to fetch the workflow specification from a remote URL.
+- Adds the possibility to fetch the workflow specification from a remote URL using the new ``/api/launch`` endpoint.
```

Comment:
Maybe we can remove this, as we already have one line for the ``/api/launch`` endpoint?